lms/static/js_test.yml

```diff
@@ -102,7 +102,6 @@ fixture_paths:
     - js/fixtures/edxnotes
     - js/fixtures/search
     - templates/search
-    - templates/courseware
```

Comment:
@chrisndodge I have added courseware templates path for the proctored exam view specs. Do not need it here anymore.